content/post/session1-startupdeepdive.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Start-up Deep Dive : Founder Requirements and Product Ideation"
+date: 2023-01-17T06:21:55+05:30
+authors:
+  - nikesh-t-v
+Description: "A participant’s take away from the first session of Startup Deep Dive, taken by Mr. Haroon Shahul on the topic \"Founder Requirements\"."

```

Comment:
Description: A participant’s take away from the first session of Startup Deep Dive, taken by Mr. Haroon Shahul on the topic "Founder Requirements"

Can you please test if removing the parent quotes altogether works fine? Or if we could use 'single quotes' for teh parent wrapper.. If so we can avoid the escape characters and that's altogether better no?

Reply:
removing the parent quotes does work fine! Nice catch

Reply:
Settt. Closing this.